server/pxf-hdfs/src/test/java/org/greenplum/pxf/plugins/hdfs/utilities/DecimalOverflowOptionTest.java

```diff
@@ -1,134 +1,27 @@
-        assertEquals(DecimalOverflowOption.DECIMAL_OVERFLOW_ERROR, parquetDecimalOverflowOption);
-    }
-
-    @Test
-    public void testDefaultDecimalOverflowOptionRound() {
-        configuration.set(PXF_ORC_WRITE_DECIMAL_OVERFLOW_PROPERTY_NAME, "round");
-        DecimalOverflowOption orcDecimalOverflowOption = DecimalOverflowOption.parseDecimalOverflowOption(configuration, PXF_ORC_WRITE_DECIMAL_OVERFLOW_PROPERTY_NAME, false);
-        assertDecimalOverflowOption(DecimalOverflowOption.DECIMAL_OVERFLOW_ROUND, orcDecimalOverflowOption);
-
-        configuration.set(PXF_ORC_WRITE_DECIMAL_OVERFLOW_PROPERTY_NAME, "ROUND");
-        orcDecimalOverflowOption = DecimalOverflowOption.parseDecimalOverflowOption(configuration, PXF_ORC_WRITE_DECIMAL_OVERFLOW_PROPERTY_NAME, false);
-        assertDecimalOverflowOption(DecimalOverflowOption.DECIMAL_OVERFLOW_ROUND, orcDecimalOverflowOption);
-
-        configuration.set(PXF_PARQUET_WRITE_DECIMAL_OVERFLOW_PROPERTY_NAME, "round");
-        DecimalOverflowOption parquetDecimalOverflowOption = DecimalOverflowOption.parseDecimalOverflowOption(configuration, PXF_PARQUET_WRITE_DECIMAL_OVERFLOW_PROPERTY_NAME, true);
-        assertEquals(DecimalOverflowOption.DECIMAL_OVERFLOW_ROUND, parquetDecimalOverflowOption);
-
-        configuration.set(PXF_PARQUET_WRITE_DECIMAL_OVERFLOW_PROPERTY_NAME, "ROUND");
-        parquetDecimalOverflowOption = DecimalOverflowOption.parseDecimalOverflowOption(configuration, PXF_PARQUET_WRITE_DECIMAL_OVERFLOW_PROPERTY_NAME, true);
-        assertEquals(DecimalOverflowOption.DECIMAL_OVERFLOW_ROUND, parquetDecimalOverflowOption);
-    }
-
     @Test
-    public void testDefaultDecimalOverflowOptionIgnore() {
-        configuration.set(PXF_ORC_WRITE_DECIMAL_OVERFLOW_PROPERTY_NAME, "ignore");
-        DecimalOverflowOption orcDecimalOverflowOption = DecimalOverflowOption.parseDecimalOverflowOption(configuration, PXF_ORC_WRITE_DECIMAL_OVERFLOW_PROPERTY_NAME, false);
-        assertDecimalOverflowOption(DecimalOverflowOption.DECIMAL_OVERFLOW_IGNORE_WITHOUT_ENFORCING, orcDecimalOverflowOption);
-
-        configuration.set(PXF_ORC_WRITE_DECIMAL_OVERFLOW_PROPERTY_NAME, "IGNORE");
-        orcDecimalOverflowOption = DecimalOverflowOption.parseDecimalOverflowOption(configuration, PXF_ORC_WRITE_DECIMAL_OVERFLOW_PROPERTY_NAME, false);
-        assertDecimalOverflowOption(DecimalOverflowOption.DECIMAL_OVERFLOW_IGNORE_WITHOUT_ENFORCING, orcDecimalOverflowOption);
-
-        configuration.set(PXF_PARQUET_WRITE_DECIMAL_OVERFLOW_PROPERTY_NAME, "ignore");
-        DecimalOverflowOption parquetDecimalOverflowOption = DecimalOverflowOption.parseDecimalOverflowOption(configuration, PXF_PARQUET_WRITE_DECIMAL_OVERFLOW_PROPERTY_NAME, true);
-        assertEquals(DecimalOverflowOption.DECIMAL_OVERFLOW_IGNORE, parquetDecimalOverflowOption);
+    public void testDecimalOverflowOptionValidness() {
```

Comment:
```suggestion
    public void testDecimalOverflowOptionValidity() {
```
I never heard `validness`, even though it is a proper dictionary word with the similar meaning, but I think `validity` is more frequently used :)